examples/GainPlugin/jsui/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "dependencies": {
     "@babel/runtime-corejs3": "^7.11.2",
-    "react-juce": "^0.2.1",
+    "react-juce": "file:../../../packages/react-juce",
```

Comment:
This is a thing that should be removed before merge. You might already know this and it's possibly left over from WIP but thought I'd add just to make sure.